tests/telemetry/events/dhcp_relay_events.py

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/env python3
+
+import logging
+import time
+from run_events_test import run_test
+from tests.common.utilities import wait_until
+
+logger = logging.getLogger(__name__)
+tag = "sonic-events-dhcp-relay"
+
+interface = ""
+ipv4_address = ""
+ipv6_address = ""
+
+
+def test_event(duthost, gnxi_path, ptfhost, data_dir, validate_yang):
+    logger.info("Beginning to test dhcp-relay events")
+    try:
+        backup_ip_address(duthost)
+    except Exception:
+        logger.info("Skipping test, no Vlan interface")
+        return
+
+    try:
+        run_test(duthost, gnxi_path, ptfhost, data_dir, validate_yang, invoke_dhcp_relay_bind_failure,
+                 "dhcp_relay_bind_failure.json", "sonic-events-dhcp-relay:dhcp-relay-bind-failure", tag)
+    finally:
+        restart_dhcp_container(duthost)
+
+
+def backup_ip_address(duthost):
+    # grab ip_addresses with subnet mask
+    interface_facts = duthost.show_ip_interface()['ansible_facts']['ip_interfaces']
+    global interface
+    interface = next((key for key in interface_facts.keys() if 'Vlan' in key), None)
+    ipv4_interface_facts = interface_facts[interface]
+    ipv6_interface_facts = duthost.interface_facts()['ansible_facts']['ansible_interface_facts'][interface]['ipv6'][0]
```

Comment:
>ipv6 [](http://example.com/codeflow?start=109&length=4)

Why ipv4 ipv6 facts are fetched in different way? Is it possible to make the fetching method similar?

Reply:
From my search on the sonic-mgmt API methods, I couldn't see such API that provided ipv4 and ipv6 addresses that map to a specific interface.

Reply:
@wangxin Do you have any suggestion?